cmplibrary/src/main/java/com/sourcepoint/cmplibrary/SpClient.kt

```diff
@@ -45,4 +45,5 @@ interface SpClient {
 
 interface UnitySpClient : SpClient {
     fun onConsentReady(consent: String)
+    fun onSpFinished(consent: String)
```

Comment:
@bohdan-go-wombat what if we migrate both of these methods to `SpClient`?

Reply:
@andresilveirah I mean we can, but what are the benefits of that?